### PR TITLE
Moving and testing ingest scripts from variantstore

### PR DESCRIPTION
**Pull request by meganshand**

### CODE REVIEW

File: scripts/variantstore_wdl/bq_ingest_arrays.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+PROJECT_ID=$1
+DATASET_NAME=$2
+STORAGE_LOCATION=$3
+TABLE_ID=$4
+if [ $5 == "true" ]; then
+  LOAD=true
+else
+  LOAD=false
+fi
```
Comment (ahaessly) [inline scripts/variantstore_wdl/bq_ingest_arrays.sh:17]: when do you not want to load?

**Review by ahaessly — APPROVED** [review]
just a couple of questions. merge whenever

File: scripts/variantstore_wdl/bq_ingest_arrays.sh (hunk 1 above)
Comment (meganshand) [reply scripts/variantstore_wdl/bq_ingest_arrays.sh:17]: My thought was that if we end up using Google Data Transfer we need a script that will create the tables but not actually load the data. Ideally we'd add the generation of the Transfers to this script too, but I didn't get around to doing that.

Now that I look at this though, I think the code for this script is all contained within the WDL and I shouldn't have committed this extra file. I'll add a comment about the Google Data Transfer to the WDL and delete this bash script.

File: scripts/variantstore_wdl/ImportArrays.wdl [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+task LoadArrays {
+  input {
+    String project_id
+    String dataset_name
+    String storage_location
+    Int table_id
+    File raw_schema
+    File sample_list_schema
+    String load = "false"
```
[the remark this answers is not in the document]
Comment (meganshand) [reply scripts/variantstore_wdl/ImportArrays.wdl:127]: I'll change it to true.

**Pull request merged**